Documentation/ceph-nfs-crd.md

```diff
@@ -25,17 +28,24 @@ metadata:
   name: my-nfs
   namespace: rook-ceph
 spec:
-  # rados property is not used in versions of Ceph equal to or greater than 
```

Comment:
How about if we separate this single example into two separate examples:
- v16.2.7: The full example, with all the recommended settings for placement and resources
- v15: The Octopus example, and no need for the placement and resources

No need for an example that works for 16.2.0-16.2.6 since we want them to move to 16.2.7 immediately, right?

Do we need to split the `deploy/example` yamls as well into `nfs.yaml` and `nfs-octopus.yaml`? Though maybe that's overkill and we should just keep the split here for readability.

Reply:
I don't see a need to split the manifests. I don't like having the extra commented-out block, but I think it's small enough and fine for now. And for any v16 version, the default config here should be good.

Reply:
Ok I see it's one small extra block, before I was thinking there were a couple blocks.